src/utils.ts

```diff
@@ -255,6 +255,8 @@ export const typify = (
       // we'll have to qualify it with the Node.js namespace.
       return 'NodeJS.ReadableStream';
   }
+  // Custom type
+  if (innerTypes) return `${typeAsString}<${typify(innerTypes[0])}>`;
```

Comment:
`innerTypes` is an array, we should technically do this for any number of inner types not just the first one. E.g. `Foo<T, R>` should be supported (I think the docs parser will read this)

Reply:
Confirmed. I tested with docs-parser and received multiple innerTypes (nice work btw!)